app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java

```diff
@@ -205,21 +214,13 @@ public void onConnected(Bundle bundle) {
     private void updateChapterPages() {
         final boolean wasOrganizer = mViewPagerAdapter != null
             && mViewPagerAdapter.getCount() == ORGANIZER_PAGES.length;
+        if (organizerCheckCallback != null) {
```

Comment:
It should be null whenever the task has finished.
On 21 Mar 2016 6:51 pm, "Said Tahsin Dane" <notifications@github.com> wrote:

> In app/src/main/java/org/gdg/frisbee/android/chapter/MainActivity.java
> <https://github.com/gdg-x/frisbee/pull/607#discussion_r56865607>:
>
> > @@ -205,21 +214,13 @@ public void onConnected(Bundle bundle) {
> >      private void updateChapterPages() {
> >          final boolean wasOrganizer = mViewPagerAdapter != null
> >              && mViewPagerAdapter.getCount() == ORGANIZER_PAGES.length;
> > +        if (organizerCheckCallback != null) {
>
> It would be great to initialize this at the beginning with wasOrganizer =
> false to make sure that it is never null.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/gdg-x/frisbee/pull/607/files/8945bc0c993f9eb28921fc4799a3b2e3808214e9#r56865607>
>